kubemarine/resources/schemas/definitions/services/loadbalancer.json

```diff
@@ -94,6 +94,30 @@
           "description": "Target https port"
         }
       }
+    },
+    "keepalived": {
```

Comment:
Should the `keep_configs_updated` option be included in that part?

Reply:
I'm not sure, that it is in scope of this changes, but in general it's good option. 
Added